### DISCUSSION TITLE
Home Screen?

### DISCUSSION THREAD

**Discussion by PhilDevProg** [flxzt/rnote#739] [Ideas]
When starting Rnote it currently creates a new document. If you open an existing document the empty document remains and you have to close its tab if you don't want it. This is really annoying in my opinion and I think a Home Screen would help with that. It would be a big empty screen with  a big button to create a document, a big button to open a document and maybe recently opened documents (maybe even with previews of them if https://github.com/flxzt/rnote/issues/53 gets implemented). This home screen disappears if another document is opened. This is just an idea and it definitely needs more discussion. It also would probably need to be an option because many people probably want to keep the current way where you just have to open Rnote and you can write instead of clicking an additional 'Create new document' button.
@flxzt What's your opinion on this?

**Comment by maymage** [comment]
The new overview in GNOME Web could serve as an initial state for Rnote, displaying selectable known files to choose from + headerbar button to start a new project. 

![grafik](https://github.com/flxzt/rnote/assets/36572014/a735e611-f5fa-4c41-b9de-438b47be580e)

**Comment by LeSnake04** [comment]
That would require some kind of preview logic with low lag.

An alternative could be a "recent"  entry in the workspace browser.
